email/README.md

```diff
@@ -0,0 +1,12 @@
+Email
+-----
+
+* Use [SendGrid] or [Amazon SES] to deliver email in staging and production
+  environments.
+* Use a tool like [ActionMailer Preview] to look at each created or updated mailer view
```

Comment:
This is probably for another PR but this is a pretty Rails specific point. It could potentially go into a Rails guide either in the base README or in /rails/email.md

Reply:
I'll try and catch things like this as part of #616 and create issues for specific changes.